website/docs/d/privatelink_endpoint.html.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: private_endpoint"
+sidebar_current: "docs-mongodbatlas-datasource-private-endpoint"
+description: |-
+    Describes a Private Endpoint.
+---
+
+# mongodbatlas_privatelink_endpoint
+
+`mongodbatlas_privatelink_endpoint` describe a Private Endpoint. This represents a Private Endpoint Connection to retrieve details regarding a private endpoint by id in an Atlas project
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find group_id in the official documentation.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_privatelink_endpoint" "test" {
+  project_id    = "<PROJECT-ID>"
+  provider_name = "AWS"
+  region        = "us-east-1"
+}
+
+data "mongodbatlas_privatelink_endpoint" "test" {
+	project_id      = "${mongodbatlas_privatelink_endpoint.test.project_id}"
+	private_link_id = "${mongodbatlas_privatelink_endpoint.test.private_link_id}"
+    provider_name = "AWS"
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) Unique identifier for the project.
+* `private_link_id` - (Required) Unique identifier of the private endpoint service that you want to retrieve.
+* `provider_name` - (Required) Cloud provider for which you want to retrieve a private endpoint service. Atlas accepts `AWS` or `AZURE`.
+
+
+## Attributes Reference
```

Comment:
I think this may be missing the Azure attributes (there are two tabs in the api docs, one with AWS and one with azure)